src/cplscheme/SerialCouplingScheme.cpp

```diff
@@ -56,10 +56,11 @@ void SerialCouplingScheme::initialize(
       int dataID = *(getAcceleration()->getDataIDs().begin());
       PRECICE_CHECK(getSendData(dataID) == nullptr,
                     "In case of serial coupling, acceleration can be defined for "
-                        << "data of second participant only!");
+                        << "data of second participant only. Ensure that the data in the <exchange tag"
+                        << " in <coupling-scheme:serial-implicit> is sent from the second participant.");
```

Comment:
Please see my comment https://github.com/precice/precice/pull/689#discussion_r422205032. This `PRECICE_CHECK` should be moved into the coupling scheme configuration, if possible.